package.json

```diff
@@ -8,7 +8,8 @@
     "lint": "lerna run lint --parallel --scope=esinstall --scope=snowpack",
     "publish": "npm run build && lerna publish --no-private",
     "format": "prettier --write '{snowpack,esinstall}/src/**/*.{ts,js}' '{test,plugins}/**/*.{ts,js}' '*.{js,json,md}' '**/*.{json,md}' '.github/**/*.{md,yml}' '!**/{_dist_,build,packages,pkg,TEST_BUILD_OUT,web_modules}/**' !test/create-snowpack-app/test-install",
-    "test": "jest --test-timeout=30000"
+    "test": "jest /test/ --test-timeout=30000",
+    "test:docs": "cd docs && yarn && yarn docs:build && jest /__test__/"
```

Comment:
nit: we can probably trim this a bit:

"cd docs && yarn docs:build && jest /__test__/"

Reply:
The docs have to follow a different pattern that the rest of the tests don’t match, hence the `/__test__/` part.

This isn’t the only solution to this problem at all but it’s the “laziest” I could think of